packages/react/src/components/AudioPlayer/README.stories.mdx

```diff
@@ -0,0 +1,124 @@
+import { Props, Description } from '@storybook/addon-docs/blocks';
+import contributing from '../../../../../docs/contributing-license.md';
+import AudioPlayer from './AudioPlayer';
+
+# Audio Player
+
+> Audio players play embedded audios using the Kaltura video platform. It can be
+> used in inline patterns. It is always the full width of its
+> containing element and maintains a fixed height of 48 pixels.
+
+> 💡 Check our
+> [CodeSandbox](https://githubbox.com/carbon-design-system/carbon-for-ibm-dotcom/tree/master/packages/react/examples/codesandbox/components/audioplayer)
+> example implementation.
+
+[![Edit @carbon/ibmdotcom-web-components](https://codesandbox.io/static/img/play-codesandbox.svg)](https://githubbox.com/carbon-design-system/carbon-for-ibm-dotcom/tree/master/packages/react/examples/codesandbox/components/audioplayer)
+
+## Getting started
+
+Here's a quick example to get you started.
+
+##### CSS
+
+```css
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+@import '@carbon/ibmdotcom-styles/scss/components/audio-player/audio-player';
+```
+
+> 💡 Only import fonts once per usage
+
+##### JS
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { AudioPlayer } from '@carbon/ibmdotcom-react';
+import 'yourapplication.scss';
+
+function App() {
+  return (
+    <AudioPlayer
+      videoId="your-video-id"
+    />
+  );
+}
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
+Add the following line in your `.env` file at the root of your project.
```

Comment:
A section can be added here regarding the environment variable needed since it's behind a feature flag. See [this](http://ibmdotcom-react-experimental.mybluemix.net/?path=/docs/components-callout-data--default#feature-flag) example.